habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1670,6 +1719,11 @@ def _step_rel_waypoint(
 
         xyt = xyt_init.copy()
 
+        # Get the current camera angle
+        goal_camera_pitch_ang = (
```

Comment:
This line can be deleted? Check my other comment below

Reply:
Hi Karmesh. The camera angle `self.camera_pitch_ang` will be updated in the `_t` loop, so this line cannot be deleted, thank you!